frontend/src/components/form/button.jsx

```diff
@@ -1,21 +1,22 @@
-import { splitProps } from 'solid-js';
-import { styled } from 'solid-styled-components';
+import { splitProps } from "solid-js";
+import { styled } from "solid-styled-components";
 
 const Button = (props) => {
-  const [ownProps, buttonProps] = splitProps(props, ['placeholder']);
+  const [ownProps, buttonProps] = splitProps(props, ["placeholder"]);
   return (
-    <div class="wrapper-input">
+    <>
       <Field {...buttonProps}>{ownProps.placeholder}</Field>
-    </div>
+    </>
   );
 };
 
 export default Button;
 
-const Field = styled('button')`
+const Field = styled("button")`
```

Comment:
Please rename it to `ButtonHTML` or something like that (maybe, search how it is done in the industry)